lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -638,27 +637,54 @@ defmodule LightningWeb.WorkflowLive.Edit do
     ~H"""
     <div
       id={"version-switcher-toggle-wrapper-#{@id}"}
-      class="flex items-center justify-between"
+      phx-click="switch-version"
+      phx-value-type="toggle"
```

Comment:
@elias-ba could you let me know why we need to repeat this click event?

Reply:
@stuartc sorry I don't understand your question. What click event are we repeating?